components/link/css/template.js

```diff
@@ -53,5 +53,5 @@ export const Link = ({
   })}"${external ? ' rel="external noopener noreferrer"' : ''}${current ? ' aria-current="page"' : ''}${
     title ? ` title="${title}"` : ''
   }${hreflang ? ` hreflang="${hreflang}"` : ''}${lang ? ` lang="${lang}"` : ''}${
-    !current ? ' rel="alternate"' : ''
+    lang && !current ? ' rel="alternate"' : ''
```

Comment:
Ik denk dat `alternate` een eigen flag moet zijn.

Reply:
Hmm of zou hreflang de check moeten zijn? Wat ik nu lees zou hreflang meestal/altijd samen komen met rel="alternate"?